### DISCUSSION TITLE
Way to group excludes

### DISCUSSION THREAD

**Discussion by ShaneCourtrille** [datalust/seq-tickets#1096] [Ideas]
In our system we have different types of logs that we want to exclude.  What I've done is create individual signals with multiple excludes.  So for example I could have "Exclude OpsMon", "Exclude Db", "Exclude Xyz" and the problem I'm running into is that we have enough of these that they take up a large chunk of the signal list.  

As is the Groupings can't be used due to their intersect behavior.  As soon as I select more than one of my excludes in a Grouping they stop working.  What would be nice is if I could click on the Grouping itself and override the default intersect behavior with the union behavior.  This way I can group things how I want and decide on case by case basis how the grouping should work.

**Comment by nblumhardt** [comment]
Thanks for the feedback, Shane. We'll give this one some thought 👍
